CLAUDE_CLI_USAGE.md

```diff
@@ -0,0 +1,160 @@
+# Claude CLI Integration
+
+This document explains how to use Claude Code CLI (claude.ai/code) as an AI provider in Task Master.
+
+## Overview
+
+The Claude CLI provider allows you to use your local Claude Code installation as an AI provider for Task Master. This is particularly useful when you want to:
+- Use Claude without consuming API credits
+- Test Task Master features with Claude
+- Use your Claude Code subscription directly
+
+## Setup
+
+1. **Install Claude Code**: Download and install Claude Code from https://claude.ai/code
+
+2. **Set the environment variable**:
+   ```bash
+   export CLAUDE_CLI_COMMAND="claude"
+   ```
+   
+   If your Claude CLI has a different name or path, adjust accordingly:
+   ```bash
+   export CLAUDE_CLI_COMMAND="/path/to/claude"
+   ```
+   
+   For MCP server usage, you can use the full path with flags:
+   ```bash
+   export CLAUDE_CLI_COMMAND="/home/user/.claude/local/claude --model opus -p --dangerously-skip-permissions"
+   ```
+
+3. **Optional: Enable file reference mode** (recommended for large PRDs):
+   ```bash
+   export CLAUDE_CLI_USE_FILE_REFERENCE="true"
+   ```
+
+4. **Configure Task Master** to use claude-cli as a provider:
+   ```bash
+   task-master models --setup
+   ```
+   
+   Then select `claude-cli` as your provider and `claude-local` as the model.
+
+## Configuration
+
+In your `.taskmaster/config.json` or `.taskmasterconfig`:
+
+```json
+{
+  "models": {
+    "main": {
+      "provider": "claude-cli",
+      "modelId": "claude-local",
+      "maxTokens": 200000,
+      "temperature": 0.2
+    }
+  }
+}
+```
```

Comment:
only mention .taskmaster/config.json from now on, lets not mention the old strategy